src/test/java/reactor/netty/udp/UdpServerTests.java

```diff
@@ -269,4 +270,27 @@ public void portBindingException() {
 
 		conn.disposeNow();
 	}
+
+	@Test(expected = UnsupportedOperationException.class)
+	public void testUdpServerWithDomainSockets() {
+		UdpServer.create()
+		         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+		         .bindNow();
+	}
+
+	@Test(expected = IllegalArgumentException.class)
+	public void testUdpServerWithDomainSocketsWithHost() {
+		UdpServer.create()
+		         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+		         .host("localhost")
+		         .bindNow();
+	}
+
+	@Test(expected = IllegalArgumentException.class)
+	public void testUdpServerWithDomainSocketsWithPort() {
+		UdpServer.create()
+		         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+		         .port(1234)
+		         .bindNow();
+	}
```

Comment:
Should there be a test for a valid server+client interaction, similar to the TCP case?

Reply:
no support for UDP protocol